Header header
uint32 channels
uint8[] connects
uint8[] data